### DISCUSSION TITLE
Nas Synology Support - Synocommunity

### DISCUSSION THREAD

**Discussion by reno3131** [autobrr/autobrr#292] [Ideas]
Add Autobrr to SynoCommunity packages to make it availlable to Nas users (non x86 nas).
Tried to do it myself but i lack competances.

https://github.com/SynoCommunity/spksrc 

https://github.com/SynoCommunity/spksrc/wiki/Developers-HOW-TO

Some package in go are supported :
https://github.com/SynoCommunity/spksrc/tree/master/cross/micro
https://github.com/SynoCommunity/spksrc/tree/master/cross/minio
https://github.com/SynoCommunity/spksrc/tree/master/cross/syncthing

Keep up the amazing work - been waiting for a autodl-irssi replacement for a long time!

Kind regards

**Comment by ludviglundgren** [comment]
Hello!

Thanks for the suggestion! I didn't even know there was a thing like this, so will take a look at the links 😄 

Thanks, will do! It's been a long time coming for sure!
